en/concepts/scene/node-component.md

```diff
@@ -11,70 +11,69 @@ Nodes are the basic building blocks of a scene. Nodes are organized in a tree-li
 
-**Before Adjustment**:
+**Before adjustment**:
 
 ![node property](scene/node-before.png)
 
-**After Adjustment**:
+**After adjustment**:
 
 ![node property](scene/node-after.png)
 
-We mentioned earlier that the component-based structure is combined to achieve functional extensions. The combination of the node and **MeshRenderer** component is shown in the following figure:
+As previously mentioned, component-based structure is combined to achieve functional extensions. The combination of the node and **MeshRenderer** component is shown in the following figure:
 
 ![node component relationship](scene/node-chart.png)
 
 ## Add Additional Components
 
-Multiple components can be added to a node to add more functionality to the node. For example:
+Multiple components can be added to a node to add more functionality to the node.
 
-We can go ahead and select the node `Sphere` in the above example, then click the **Add Component** button at the bottom of the **Inspector** panel and select **Light -> DirectionalLight** to add a **Directional Light** component.
+For example, select the node `Sphere` in the above example, then click the **Add Component** button at the bottom of the **Inspector** panel and select **Light -> DirectionalLight** to add a **Directional Light** component.
 
-Then set the properties of the **Directional Light** component, e.g. adjust the `Color` property of the directional light to red, and you can see that the color of the sphere model has changed, i.e. the **DirectionalLight** component we added to the node has taken effect!
+Next, set the properties of the **Directional Light** component, e.g. adjust the `Color` property of the directional light to red, and notice that the color of the sphere model has changed, i.e. the **DirectionalLight** component that was added to the node has taken effect!
 
 ![button property](scene/directional-light.png)
 
-> Here is just a brief example of a more obvious effect. It is not recommended to add a **DirectionalLight** component to a sphere node.
-
-For more details about other components, please refer to [components](../../editor/components/index.md) documentation.
-
-## Summary
-
-In the above example, we first combine the **MeshRenderer** component with the node, and with the mesh resource that can specify the rendering material, we can modify the node properties to display the model in different ways, such as scaling and rotation. Now we add the **DirectionalLight** component to the node so that the node can show different effects depending on the state of the directional light source. This is the workflow of Cocos Creator 3.0 component-based development, where we can combine different functions on a node in such a way that we can achieve more complex goals.
```

Comment:
这部分内容怎么删掉了呢

Reply:
I deleted it because it was wordy and didn't state anything that wasn't already talked about. Lots of other documents don't have a summary section at the end. Why does this one? We should add a summary section for all documents then.

Reply:
OK